nrf_modem/doc/CHANGELOG.rst

```diff
@@ -9,16 +9,44 @@ Changelog
 
 All notable changes to this project are documented in this file.
 
-nrf_modem
-*********
+nrf_modem 3.0.0
+***************
```

Comment:
The version would be added in the doc version PR is what we had agreed. @lemrey - Could you confirm this?

Reply:
Alright👍, will update